__tests__/DefaultApi.spec.ts

```diff
@@ -455,5 +455,42 @@ describe('DefaultAPI', () => {
         let data = await apiInstance.listConferenceRecordings(callId, conferenceId, dateCreated)
         expect(data).toBeInstanceOf(RecordingList)
     });
+
+    test('#getNextPage', async () => {
```

Comment:
is this generated or handwritten?

Reply:
Hand written, I don't think there's a generator file for this test file

Reply:
I wonder if we need more tests for this getNextPage function similar to what's in the Python SDK for each request that can have a paginated response or if having this one test is sufficient enough that the getNextPage function works?

We can also create a ticket to look into whether we can auto-generate the default api test file since currently this is being manually maintained.

Reply:
if we aren't generating this test file then it seems like we should just leave it as is, would be a big lift to write out all of those scenarios manually right now, and would be hard to track that we need to add a manual test case if more list operations are added in the future